### PR TITLE
Components

### PR DESCRIPTION
**Pull request by collins-kiprotich**
Add a calculator Component UI to my App

### CODE REVIEW

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Collins, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- linters are passing

## Required Changes ♻️
- [ ] there are stills some files in the public folder created by the CRA which need to be removed.
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1,70 +1,169 @@
-# Getting Started with Create React App
+<a name="readme-top"></a>
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
```
Comment (danmainah) [inline README.md:6]: - [ ] [ioptional] It would be nice if you removed this logo or added your own logo

File: README.md [hunk 2]
```diff
@@ -1,70 +1,169 @@
...
+
+👤 **Collins Kiprotich**
+
+- GitHub: [Collins-kiprotich](https://github.com/collins-kiprotich)
+- Twitter: [@jcee](https://twitter.com/Jcee01188383)
+- LinkedIn: [Collins Kiprotich](https://www.linkedin.com/in/collins-kiprotich-152b52246/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```
Comment (danmainah) [inline README.md:138]: - [ ] [optional] @collins-kiprotich It would be nice if you added future features in your readme file

**Review by danmainah — APPROVED** [review]
Hi @collins-kiprotich , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️ No linter errors
✔️ Gitflow was correctly used
✔️  Good design

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**